doc/classes/Signal.xml

```diff
@@ -85,7 +85,10 @@
 		<method name="get_connections" qualifiers="const">
 			<return type="Array" />
 			<description>
-				Returns the list of [Callable]s connected to this signal.
+				Returns an [Array] of connections for this signal. Each connection is represented as a [Dictionary] that contains three entries:
+				- [code]signal[/code] is a reference to this signal;
+				- [code]callable[/code] is a reference to the connected [Callable];
+				- [code]flags[/code] is a combination of [enum Object.ConnectFlags].
```

Comment:
Maybe it's worth clarifying that it's an integer.

Reply:
Was about to make a change, but this same phrase is used for a few other Object methods. I think we can wait and see if there is any confusion around what flags are, then we can rephrase all relevant sentences.